bip-0370.mediawiki

```diff
@@ -103,7 +103,7 @@ The new global types for PSBT Version 2 are as follows:
 | None
 | No key data
 | <tt><8-bit uint></tt>
-| An 8 bit little endian unsigned integer as a bitfield for various transaction modification flags. Bit 0 is the Inputs Modifiable Flag and indicates whether inputs can be modified. Bit 1 is the Outputs Modifiable Flag and indicates whether outputs can be modified. Bit 2 is the Has SIGHASH_SINGLE flag and indicates whether the transaction has a SIGHASH_SINGLE signature who's input and output pairing must be preserved. Bit 2 essentially indicates that the Constructor must iterate the inputs to determine whether and how to add an input.
+| An 8 bit little endian unsigned integer as a bitfield for various transaction modification flags. Bit 0 is the Inputs Modifiable Flag and indicates whether the inputs can be added or removed. Bit 1 is the Outputs Modifiable Flag and indicates whether outputs can be added or removed. Bit 2 is the Has SIGHASH_SINGLE flag and indicates whether the transaction has a SIGHASH_SINGLE signature who's input and output pairing must be preserved. Bit 2 essentially indicates that the Constructor must iterate the inputs to determine whether and how to add an input.
```

Comment:
> determine whether and how to add an input

suggest `determine whether and how to add or remove an input'

Reply:
> An 8 bit little endian unsigned integer 

Suggest `An 8 bit unsigned integer`, there is no such thing as a big/little endian byte.

Reply:
The bits do not specify a how, so that would be incorrect to state.

Fixed the endian thing.

Reply:
The "how" part is already in the text which says: "to determine whether **and how** to add an input".

I'm suggesting changing `to add` into `to add or remove`. If you think `and how` should be removed no objection here.

Reply:
Ah, I see. I thought you were talking about a different section. Changed that.